rotkehlchen/api/rest.py

```diff
@@ -3072,3 +3072,34 @@ def get_user_binance_pairs(self, name: str, location: Location) -> Response:
             ),
             status_code=HTTPStatus.OK,
         )
+
+    def _perform_add_manual_price(self, historical_price: HistoricalPrice) -> Dict[str, Any]:
+        # pylint: disable=no-self-use
+        GlobalDBHandler().add_historical_prices([historical_price])
```

Comment:
This is too simple. You will need to create a version of `add_historical_prices()` that accepts only one historical price and raises an error if something goes wrong. For example if there is already another price at the same timestamp.